src/api/routes/v1/get-network.ts

```diff
@@ -0,0 +1,237 @@
+import { Request, Response } from 'express'
+
+import Crawler from '../../../crawler/crawl'
+import crawlNode from '../../../crawler/network'
+import { query } from '../../../shared/database'
+import { Network } from '../../../shared/database/networks'
+import { Crawl } from '../../../shared/types'
+import logger from '../../../shared/utils/logger'
+
+const log = logger({ name: 'api-get-network' })
+
+const CRAWL_PORTS = [51235, 2459, 30001]
+
+let maxNetwork: number
+
+async function getMaxNetwork(): Promise<void> {
```

Comment:
`getMaxNetwork` implies that it will return maxNetwork, I think this name better reflects it's behavior.
```suggestion
async function updateMaxNetwork(): Promise<void> {
```